api/src/main/java/org/pmiops/workbench/mail/MailService.java

```diff
@@ -14,9 +14,6 @@ void sendWelcomeEmail(final String contactEmail, final String password, final Us
   void sendInstitutionUserInstructions(final String contactEmail, final String userInstructions)
       throws MessagingException;
 
-  void sendBetaAccessCompleteEmail(final String contactEmail, final String username)
```

Comment:
no longer used.  There's one more beta access email that has a code reference.  That will be removed in RW-6913.